src/systemctl/systemctl-show.c

```diff
@@ -433,6 +433,13 @@ static void print_status_info(
                                FORMAT_TIMESTAMP_STYLE(until_timestamp, arg_timestamp_style),
                                FORMAT_TIMESTAMP_RELATIVE(until_timestamp));
                 }
+
+                if (STRPTR_IN_SET(i->active_state, "inactive", "failed")) {
+                        usec_t duration;
+
+                        duration = i->active_exit_timestamp - i->active_enter_timestamp;
+                        printf("   Duration: %s\n", FORMAT_TIMESPAN(duration, USEC_PER_SEC));
```

Comment:
Is the `USEC_PER_SEC` correct here? I'm confused on how the accuracy argument for FORMAT_TIMESPAN works

Reply:
well depends. the question is what level of accuracy would you expect people want to see here? I figure msec accuracy would be good, hence would probably pass MSEC_PER_SEC.